v5_MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,42 @@
+# Migrating to Unleash-Client-Dotnet 5.0.0
+
+This guide highlights the key changes you should be aware of when upgrading to v5.0.0 of the Unleash client.
+
+## Custom strategy changes
+
+Custom strategies no longer provide the option to access the constraints in their interface. The method `bool IsEnabled(Dictionary<string, string> parameters, UnleashContext context, IEnumerable<Constraint> constraints)` no longer exists. Other than that, custom strategies remain unchanged.
+
+## Direct access to feature toggles
```

Comment:
We also removed `GetVariants`

Reply:
Good spot, thank you!

Reply:
Made a note in [last commit](https://github.com/Unleash/unleash-client-dotnet/pull/257/commits/b199365e9352c293b7b128c5fe863bc5b8073e66)